### PR TITLE
DDF-3260 Upgrade wss4j 

### PR DESCRIPTION
**Pull request by vinamartin**
#### What does this PR do?
Upgrades wss4j + removes this change https://github.com/codice/ddf/pull/2179/commits/eaa40e59d8675a669a18c1e3e8f5b1ebd154a31a which is fixed in the wss4j version we're upgrading to. 

#### Who is reviewing it? 
@clockard @emmberk @jlcsmith 
 
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@coyotesqrl 
@stustison 

#### What are the relevant tickets?
[DDF-3260](https://codice.atlassian.net/browse/DDF-3260)
[DDF-3187](https://codice.atlassian.net/browse/DDF-3187)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: platform/security/sts/security-sts-realm/pom.xml [hunk 1]
```diff
@@ -44,6 +44,11 @@
             <artifactId>security-core-api</artifactId>
         </dependency>
         <dependency>
+            <groupId>ddf.security</groupId>
```
Comment (clockard) [inline platform/security/sts/security-sts-realm/pom.xml]: Good catch!

**Comment by vinamartin** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by emmberk** [comment]
@vinamartin reminder to update the waitForReady command

**Comment by vinamartin** [comment]
build now

**Pull request merged**